js/shop.js

```diff
@@ -0,0 +1,219 @@
+// allowed_terms is a whitelist of Canvas enrollment_term_ids where shopping is allowed
+// NOTE - the term ids in allowed_terms must be strings, not ints
+var allowed_terms = ["69"];
```

Comment:
Same question as in the colgsas PR. Is 69 the right term to be checked in or should it be blank?